src/piadm.sh

```diff
@@ -109,13 +124,10 @@ piname_present_get_bootfs() {
 	    usage
 	fi
     fi
-
-    poolpresent $pool
-    echo $bootfs
 }
 
-# Use "-k" for now until we ship CAs with the Platform Image again.
-CURL="curl -k"
+# Defined as a variable in case we need to add parameters (like -s) to it.
+CURL="curl -s"
 
 # Well-known source of SmartOS Platform Images
 URL_PREFIX=https://us-east.manta.joyent.com/Joyent_Dev/public/SmartOS/
```

Comment:
Not necessary in v1, but maybe consider making this a conditional assignment (e.g. `DEFAULT_URL_PREFIX=https://us-east....; URL_PREFIX=${URL_PREFIX:-${DEFAULT_URL_PREFIX}}`) to allow people to be able to specify a custom URL in a future change.

Reply:
Good idea, I think I'll do that now (but leave it undocumented).